packages/manager/modules/pci/src/projects/projects.routing.js

```diff
@@ -52,13 +60,22 @@ export default /* @ngInject */ ($stateProvider) => {
 
         return promise;
       },
+
       guideUrl: () => GUIDES_URL,
+
+      isValidHdsSupportLevel: () => {
+        const { supportLevel } = Environment.getUser();
+        const sl = new SupportLevel(supportLevel);
```

Comment:
**thought**: Maybe the supportLevel returned by `Environment.getUser()` should already be an instance of `SupportLevel`

Reply:
No, i tested it, it's a JS object.

Reply:
I don't doubt it, but maybe we should when using `setUser()` on Environment convert the supportLevel to a `SupportLevel` instance

Reply:
fixed